Sources/Controllers/TargetMenu/GPX/TrackMenu/OATrackMenuHudViewController.mm

```diff
@@ -2044,6 +2093,59 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         }
         outCell = cell;
     }
+    else if ([cellData.type isEqualToString:[OAImageDescTableViewCell getCellIdentifier]])
+    {
+        OAImageDescTableViewCell* cell = [tableView dequeueReusableCellWithIdentifier:[OAImageDescTableViewCell getCellIdentifier]];
+        if (cell == nil)
+        {
+            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OAImageDescTableViewCell getCellIdentifier] owner:self options:nil];
+            cell = (OAImageDescTableViewCell *)[nib objectAtIndex:0];
+        }
+        cell.selectionStyle = UITableViewCellSelectionStyleNone;
+        cell.separatorInset = UIEdgeInsetsMake(0., DBL_MAX, 0., 0.);
+        cell.iconView.contentMode = UIViewContentModeScaleAspectFill;
+        cell.descView.hidden = YES;
+        cell.imageBottomToLabelConstraint.priority = 1;
+        cell.imageBottomConstraint.priority = 1000;
+        cell.imageBottomConstraint.constant = 0;
+        NSString *imageUrl = cellData.values[@"img"];
+        
+        cell.iconView.image = nil;
+        if (_isImageDownladFinished)
+        {
+            cell.activityIndicatorView.hidden = YES;
+            cell.iconView.image = _cachedImage;
+            if (_cachedImage)
+            {
+                cell.imageTopConstraint.constant = 16;
+                cell.iconViewHeight.constant = 149;
+            }
+            else
+            {
+                cell.imageTopConstraint.constant = 1;
+                cell.iconViewHeight.constant = 1;
+            }
+        }
+        else
+        {
+            cell.activityIndicatorView.hidden = NO;
+            [cell.activityIndicatorView startAnimating];
+            dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0ul);
+            dispatch_async(queue, ^{
```

Comment:
1. Simply use 
```
dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0), ^{
});
```
2. NEVER put async download code into cellForRow, ALWAYS use cache to avoid pointless reloading from the internet on scroll.